client/main/notification.js

```diff
@@ -0,0 +1,65 @@
+
+'use strict';
+
+const _ = require('underscore');
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.$ = $;
+var SilkyView = require('./view');
+
+var Notification = SilkyView.extend({
+    className: "notification",
+    events : {
+        'click .silky-notification-link'    : '_linkClicked'
+    },
+    initialize: function() {
+        this.model.on("change", this.render, this);
+        this.render();
+    },
+    render: function() {
+
+        this.$el.addClass("silky-notification");
+        
+        if (this.model.attributes.visible){
+            this.$el.css("visibility", "visible");        
+        }else{
+            this.$el.css("visibility", "hidden");
+        }
+        
+        if (this.model.attributes.timeToFade > 0){
+            window.setTimeout(_.bind(function(){
```

Comment:
we've started using arrow functions for things like this. do you know about arrow functions? so this

    window.setTimeout(_.bind(function(){
       ...
    }, this), time);

becomes:

    window.setTimeout(() =>  {
        ...
    }, time);

they're part of ES6